pkg/content/parsing.go

```diff
@@ -16,11 +17,11 @@ import (
 )
 
 type PageMeta struct {
-	Categories []string `toml:"categories"`
-	Date       string   `toml:"date"`
-	Redirect   string   `toml:"redirect"`
-	Root       bool     `toml:"root"`
-	YoutubeId  string   `toml:"youtube_id"`
+	Categories []string   `toml:"categories"`
+	Date       *time.Time `toml:"date"`
```

Comment:
made more since as a pointer so it can be nil, the default time.Time isn't falsey